be/src/user/dto/user.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsNotEmpty, IsEmail } from 'class-validator';
+import { Role } from '../roles/user.roles';
+
+export class UserDTO {
```

Comment:
We don't need a UserDto, just use the original entity and add fields to it

Reply:
NO NO NO, This will pose security risks, password hashes can be obtained from queries then.

Reply:
Just don't add a field to the pw? Also create a test that checks that the pw is not exposed, then nobody will add it by accident as well.

Reply:
Forgot to add test, adding that in a separate commit